F3FChrono/data/Round.py

```diff
@@ -88,8 +87,10 @@ def next_pilot(self, insert_database=False):
         return self.get_current_competitor()
 
     def next_pilot_database(self):
-        if self._current_competitor_index < len(self._flight_order) - 1:
-            self._current_competitor_index += 1
+        nb_run=len(self.groups[len(self.groups)-1].runs)
```

Comment:
self.groups[len(self.groups)-1] could be simplified by self.groups[-1]

Reply:
done.
I forget this trick